pkg/pool-stable/test/foundry/E2eErc4626Swap.t.sol

```diff
@@ -0,0 +1,58 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Should this filename be "Swaps" for consistency with "E2eErc4626Swaps.t.sol"? (Or make them both Swap)?

Reply:
Done!